src/js/queryService.js

```diff
@@ -348,11 +377,12 @@ export const sendSuggestion = ({
   apiKey,
   domain,
   token,
+  isQandA,
 }) => {
   const url = `${domain}/autoql/api/v1/query/${queryId}/suggestions?key=${apiKey}`
   const data = { suggestion }
 
-  if (!token || !domain || !apiKey) {
+  if (!isQandA && (!token || !domain || !apiKey)) {
```

Comment:
Maybe out (!token || !domain || !apiKey) into a var and then use in the var so the If statement isnt to large. Just a suggestion though.

Reply:
Interesting suggestion, for this case I don't really feel like it's necessary as all of the other functions follow this format.